NeosModLoader/NeosModLoader.csproj

```diff
@@ -42,7 +42,6 @@
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
-    <Reference Include="System.Data.DataSetExtensions" />

```

Comment:
What's up with this?

Reply:
i accidentally removed the reference and it still compiled so i did not think much of it, if you would like im happy to add that reference back.

Reply:
I'm curious what the reason for removing it was? Is it incompatible with Linux somehow?